modules/platforms/cpp/CMakeLists.txt

```diff
@@ -24,18 +24,61 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_PROJECT_VERSION ${PROJECT_VERSION})
 
 set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_BINARY_DIR})
+list(APPEND CMAKE_PREFIX_PATH ${CMAKE_BINARY_DIR})
 
-if(EXISTS ${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-    message(STATUS "Conan detected. Enabling...")
-    include(${CMAKE_BINARY_DIR}/conan_paths.cmake)
-    include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-    conan_basic_setup()
+include(conan)
+
+get_property(is_multi_config GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
+
+# Install required libraries.
+if (is_multi_config)
+    # These settings are for multi-config generators like Visual Studio.
+    message(STATUS "Using a multi-config generator")
+
+    conan_cmake_configure(REQUIRES msgpack-c/4.0.0 gtest/1.12.1
+                          GENERATORS cmake_find_package_multi)
```

Comment:
Let's move a list of libraries to a separate variable.

Reply:
Done.